Automattic/pocket-casts-android
# Transcripts - Support HTML type

## Description

This supports HTML transcript type. 

## Testing Instructions
1. Play an episode that supports HTML transcript by default e.g. `If books could kill` (https://pca.st/tqib657g)
2. Open full-screen player
3. Tap on transcript bottom shelf icon
4. ✅ Notice that HTML content is shown in the transcript view

## Screenshots or Screencast 

https://github.com/user-attachments/assets/06622741-849e-4c4f-90d6-62e829926e54


## Checklist
 
#### I have tested any UI changes...
